airbyte-workers/src/main/java/io/airbyte/workers/process/KubeProcessBuilderFactory.java

```diff
@@ -86,7 +87,7 @@ public ProcessBuilder create(String jobId, int attempt, final Path jobRoot, fina
               "--restart=Never",
               "--overrides=" + overrides, // fails if you add quotes around the overrides string
               podName);
-
+      // TODO handle entrypoint override (to run DbtTransformationRunner for example)
```

Comment:
You'll likely need to override `cmd` and `args` in the pod's spec in `kube_runner_template.yaml`

Reply:
I opened this issue for now for kube version of this: https://github.com/airbytehq/airbyte/issues/3441